elements/pfe-cta/src/pfe-cta--lightdom.scss

```diff
@@ -1,191 +1,11 @@
-      )};
-      outline: none;
-      position: absolute;
-      top: 2px;
-      left: 2px;
-      bottom: 2px;
-      right: 2px;
-      border-radius: 2px;
-    }
-
-    .pfe-cta--arrow {
-      fill: #{pfe-local(Color--focus)};
-    }
-  }
-}
-
-// targeting default cta's
-.pfe-cta--wrapper {
-    padding: 0 !important;
-}
-
-// removing padding from wrapper and placing it on link
-[pfe-priority] .pfe-cta--wrapper {
-    padding: 0 !important;
-}
-
-[class^="pfe-cta"][class="PFElement"][pfe-priority],
-[pfe-priority] .pfe-cta--wrapper {
-  a,
-  input {
-    text-decoration: none;
-    padding: #{pfe-local(
-        $cssvar: Padding, $fallback: (16px 32px),
-    )} !important;
   }
 }
-
-[class^="pfe-cta"][class="PFElement"][pfe-priority="secondary"], //ie11
-[pfe-priority="secondary"] .pfe-cta--wrapper {
-  input,
-  button,
-  a {
-      color: #{pfe-local(
-          $cssvar: Color, $fallback: #{pfe-color(text)}
-      )} !important;
-  }
-
-  &:hover input,
-  &:hover button,
-  &:hover a,
-  input:hover,
-  button:hover,
-  a:hover {
-      color: #{pfe-local(
-          $cssvar: Color--hover, $fallback: #{pfe-color(link--on-saturated)}
-      )} !important;
-  }
-  @include browser-query(edge) {
-      a:focus {
-          color: #{pfe-local(
-            $cssvar: Color--focus, $fallback: #{pfe-color(link--focus)},
-          )} !important;
-      }
-    }
-}
-
-@include browser-query(edge) {
-  .pfe-cta--arrow.pfe-cta {
-      top:    .25rem;
-      width:  18px;
-      height: 18px;
-  }
-}
-
-@include browser-query(ie11) {
-  [class^="pfe-cta"] .pfe-cta--arrow.pfe-cta {
-      top:    .35rem;
-      width:  16px;
-      height: 20px;
-  }
-  pfe-cta.PFElement{
-    button {
-      background-color: transparent !important;
-      border: none !important;
-      margin: 0 !important;
-      padding: 0 !important;
-    }
-    &[pfe-priority="primary"] {
-      a, a:visited, a:focus, a:hover {
-        color: #{pfe-local(
-          $cssvar: Color, $fallback: #{pfe-color(link--on-saturated)},
-        )} !important;
-      }
-    }
-    &[pfe-priority="primary"][pfe-color="lightest"],
-    &[pfe-priority="secondary"][pfe-color="lightest"] {
-        border-color: #{pfe-local(
-          $cssvar: BorderColor, 
-          $fallback: #{pfe-color(surface--darkest)},
-        )} !important;
-        a {
-            color: #{pfe-local(
-              $cssvar: Color, 
-              $fallback: #{pfe-color(text)},
-            )} !important;
-        }
-      a:hover {
-          color: #{pfe-local(
-            $cssvar: Color, 
-            $fallback: #{pfe-color(ui-complement)},
-          )} !important;
-      }
-    }
-    &[pfe-priority="secondary"][pfe-variant="wind"] {
-      a, a:visited, a:focus, a:hover {
-          color: #{pfe-local(
-            $cssvar: Color, 
-            $fallback: #{pfe-color(link)},
-          )} !important;
-      }
-    }
-  }
-}
-
-// Fallback styles for safari to read a border on focus
```

Comment:
Hmm, some of these [styles were just added](https://github.com/patternfly/patternfly-elements/pull/665/files#diff-78f9162f9b923c73f4b48c8945eff53bR180) because we were not seeing correct focus states in IE, Edge & Safari 
Is this branch pushed to a sandbox?

Reply:
You can view the styles here without needing to push to a sandbox: https://deploy-preview-630--happy-galileo-ea79c4.netlify.com/elements/pfe-cta/demo/

The light dom styles aren't needed since I altered the approach to how the border appears.  Instead of using pseudo elements which was causing issues in Safari, I used a span tag with the same styles and added z-index to ensure the link layers on top.  This lets us delete almost the entire light dom file!  Check it out.  I tested it really extensively to ensure the approach was valid and we shouldn't need that extra code anymore.

Reply:
Whoa! Sorry the pfe-cta.scss diff was collapsed so I missed it while skimming... but wow, I see what you mean now :D  thanks for adding the netlify link too!